test/swmr.c

```diff
@@ -1554,20 +1554,26 @@ test_metadata_read_retry_info(hid_t in_fapl)
  *      Should fail to enable SWMR as the file is already in SWMR writing mode
  *      Close the file
  *
- *  (B) Opening a file
- *      Open the file with write + non-latest-format
- *      --file has v3 superblock and all latest version support enabled
- *      Open dataset "dataset1" 3 times--keep it open
- *    Write to "dataset1"
- *    Create a group in the file
- *      Create a chunked dataset "dataset2" in the group--should be using latest chunk indexing--keep it open
- *      Should succeed in enabling SWMR
- *    Should succeed in reading from multiple opens of "dataset1"
- *      Close multiple opens of "dataset1"
- *    Close "dataset2"
- *      Create "dataset3"--should be using latest chunk indexing
- *      Close "dataset3"
- *      Close the group and file
+ *  (B) Open the file with write + non-latest-format (earliest, latest)
+ *      --file has v3 superblock
+ *      Create a group in the file
+ *      Create a chunked dataset "dataset2" in the group--keep it open
+ *      Verify "dataset2" is using v1 btree indexing
+ *      Should fail to enable SWMR writing
```

Comment:
I thought the idea behind the libver bounds was to try and use earlier versions for file format compatibility, but to "fallback" to later versions if necessary to complete the requested operations. Is this a violation of that principle?

Reply:
If I understand the question correctly:  this test is for testing H5Fstart_swmr_write() which requires V1.10 version 3 superblock to begin with.

Reply:
I was trying to understand the significance of the changes to libver bounds handling. I think I was confusing object creation versus later operations on those objects. I guess it makes sense for SWMR to fail if old objects exist that it can't handle.